Client/Clique/Common/Models/Event.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Event.swift
+//  Clique
+//
+//  Created by Infinum on 15.11.2022..
+//
+
+import Foundation
+
+struct Event: Codable {
+    let eventID: Int
+    let eventName: String
+    let eventLocation: String
+    let eventTimestamp: String
+    let eventParticipantNumber: Int
+    let eventCost: Double?
+    let eventCurrency: String?
+    let eventCreator: User
+    let eventCategory: String
+    let eventDescription: String?
```

Comment:
also you have Event struct here. And you are declaring every param with event prefix. Please do not use this practice, it looks better and it is less code if it is `event.id` instead of `event.eventId`